packages/@o3r/schematics/src/utility/modules.ts

```diff
@@ -104,11 +104,15 @@ export function getMainFilePath(tree: Tree, context: SchematicContext) {
  * one of its modules.
  *
  * @param tree
+ * @param options @see RuleFactory.options
```

Comment:
it is not supposed to be `{@see RuleFactory.options}` ?

Reply:
I've copy pasted what we already have everywhere else
if we want to update that everywhere this should be done in a separate PR